invenio_rdm_records/fixtures/data/vocabularies/affiliations_ror.yaml

```diff
@@ -5,78 +5,123 @@
     de: "\_Europ\xE4ische Organisation f\xFCr Kernforschung"
     en: European Organization for Nuclear Research
     fr: "Organisation europ\xE9enne pour la recherche nucl\xE9aire"
+  identifiers:
+    - scheme: ror
+      identifier: 01ggx4157
```

Comment:
I think we need the full ROR ``https://ror.org/01ggx4157``

https://ror.org/display-guidelines/

Reply:
We could, I was just trying not to duplicate/mix content of ID and URI... If you see in the screenshots above, it is displayed correctly. Therefore, the guidelines are met.

If we want a more in depth fix, we should do some refactor as mentioned in this PR description:
- Add `scheme` to IdentifierScheme in Marshmallow-Utils
- Modify loading to add the `id + scheme` from the current `subtype` as each affiliation identifier. This would mean removing the current yaml changes.

Reply:
So ROR website is not really clear about what is considered the ID. Initially, it seemed to me that they considered the entire URL as the ID, hence we should store the entire ID. I've tried to re-read everything and there's not a clear definition, so let's go with the short form.